src/programmable_signals_gui.cpp

```diff
@@ -511,7 +511,12 @@ class ProgramWindow : public Window {
 		this->RaiseWidgetWhenLowered(PROGRAM_WIDGET_COND_SET_SIGNAL);
 	}
 
-	virtual void OnQueryTextFinished(std::optional<std::string> str) override
+	virtual void OnQueryTextFinished(std::optional<std::string> str) override final
+	{
+		OnQueryTextFinished(str, {});
```

Comment:
Should these be std::nullopt?

Reply:
I think it makes no difference to the generated code whether the parameter is {} or std::nullopt, and I don't find either one particularly easier to read over the other. So, for me, it's a six of one, half a dozen of the other situation.